alacritty.yml

```diff
@@ -83,8 +83,8 @@
   # Specifying '0' will disable scrolling.
   #history: 10000
 
-  # Number of lines the viewport will move for every line scrolled when
-  # scrollback is enabled (history > 0).
+  # Acceleration for mouse scrolling, number of lines the viewport will move for
+  # every line scrolled when scrollback is enabled (history > 0).
```

Comment:
This is still not entirely true, I can move by 1 line with `multiplier: 10`, it's acceleration for mouse scrolling events in short, but I don't have good wording here.

Reply:
How about just:
"Mouse scrolling acceleration"

Reply:
I guess it's better, but I still think that it could be strange wording. what do you think @chrisduerr ?

Reply:
Not a huge fan, since this doesn't correctly represent that this is more than just an abstract number. Acceleration also sounds like mouse acceleration, so bigger movements, bigger outcome. But thankfully that's not the case (at least on Linux/Windows).

I'd suggest maybe something like the following?

```
Modifier for multiplying the mouse scrolling distance.
```

I'd definitely stay away from the word acceleration at least.